### DISCUSSION TITLE
thoughts and reactions to name choice

### DISCUSSION THREAD

**Discussion by castedo** [singlesourcepub/community#51] [Ideas]
Does anyone have thoughts or reactions regarding the following naming? 

I hope the following name will help communication about the specific kind of single-source publishing that I'm working on. In my use case, JATS XML (with additional binary data) is permanently and publicly archived. This can be rendered into PDF files and/or web page content via existing and future independent applications and services. This rendering is fully automated with free open-source software (FOSS).  I'm not thinking of format name but rather a name for an archived digital object of this type.

So what is a good name choice for the concrete binary/digital representation of article/manuscript/paper content publicly and permanently archived from which others can freely generate HTML content and PDF files using FOSS? 

Here's my short list of candidates (in alphabetical order) riffing off "preprint":

* baseprint
* bitprint
* digiprint
* interprint
* uniprint
* printpack

I'll also create a github poll tomorrow or this weekend and link here if people want to just vote. But thoughts and reactions are also helpful to know, and feel free to suggest other names.

PROOF-OF-CONCEPT EXAMPLE
------------------------

An example of permanently and publicly archived JATS XML is:
https://archive.softwareheritage.org/browse/content/sha1_git:aedda1c6f4b9bdba9dfeadea9b9677532fb055e0/
which is automatically rendered with FOSS into:
https://popgen.es/0iV47kWzQAuyONrIDG538k3x3Qc/0.2/
and
https://popgen.es/0iV47kWzQAuyONrIDG538k3x3Qc/0.2/article.pdf

**Comment by ronste** [comment]
Hi @castedo ,

from what I read above I see two major topics your works builds upon, archiving and rendering into output formats. The latter is covered in your list above. What about archiving? Or is this not a central point?

**Reply by castedo** [reply]
I think archiving is an important point because it adds very severe constraints on a "foobarprint" storage format. For instance, the defacto usage of `specific-use=""` is NOT OK for a foobarprint. In typical JATS XML usage `specific-use` is fine, such as your `<table specific-use="Tabelle-gedreht-rechts"` use case, but not OK for a foobarprint. My rough impression to date is that the JATS XML spec and defacto usage is full of custom tweaks that are specific to particular document processing system.

I suspect part of the usefulness of being able to talk about a "foobarprint" is so we can say things like `specific-use=` is fine for JATS XML but it should not be in the standard format for a foobarprint ... in large part because of this archiving constraint.

I also brainstormed "archiprint" and similar. But I fear a reader might jump to the erroneous conclusion that it is preprint in archaeology.

Maybe there is a cool German prefix or wording that works here? I love naming like "eigenvalue", "eigenvector", "eigenspectrum". Germans have such a cool language; English should borrow more morphemes from German, not just its grammar! :smile:

**Reply by tarleb** [reply]
“Grundpaper” :wink:

**Comment by tarleb** [comment]
I like the idea of giving a name to that. “Baseprint” is my favorite, it conveys the intended meaning nicely!
